tasks.py

```diff
@@ -136,7 +136,7 @@ def _detect_space(branch=None, yes=False):
 
 DEPLOY_RULES = (
     ('prod', _detect_prod),
-    ('stage', lambda _, branch: branch == 'master'),
+    ('stage', lambda _, branch: branch == 'release'),
```

Comment:
I wrote up the associated deploy workflow in #942 but still have a few questions. If we're happy with that workflow (assuming I described it correctly), then I agree with your revision. Let's resolve #942 and then make and merge that change.